### ISSUE TITLE
Confusion about some table comparison methods？

### ISSUE THREAD

**Issue by yanglebupt** [RyanWangZf/MediTab#2]
Your work is very exciting! Congratulations on your work being accepted and recognized! I raise this question out of curiosity about comparative experiments and multi-datasets training method. I apologize for disturbing you!

![image](https://github.com/user-attachments/assets/3be9dace-bcf4-46e7-8494-dc771348eb69)

`Note that we train a single classifier for MediTab and predict on all datasets, while the base-lines need to be trained on each dataset separately`. This very short sentence seems to lack clarity in expression, so I have some simple questions troubling me:

- In `Table3/4`, there are `9 Patient Outcome Datasets/3 Clinical Trial Outcome Datasets`. Which dataset was used to `train a single classifier for MediTab`? Or using all datasets in each task?

`We assess the zero-shot prediction capability of MediTab on two tasks. For the evaluation of the dataset D, we deliberately exclude D from the training data during step 2, where pseudo labels are generated for the external database.`

- If use one dataset in training, Where did the exclusion come from?

And in Figure 2, This seems to be using all the datasets in a task for training. That is to say, for the Patient Outcome task, using all 9 datasets, for Clinical Trial Outcome Task use all 3 datasets? Is it unfair to compare xgboost trained on a single dataset in this way, even though xgboost can not utilize multiple datasets?

![image](https://github.com/user-attachments/assets/b1663a51-c5ab-480f-9084-057d9dcecac0)

If you could spare some time to answer my questions, I would be extremely grateful!
